cfme/services/myservice/__init__.py

```diff
@@ -34,6 +35,19 @@ def __init__(self, appliance, name=None, description=None, vm_name=None):
         self.vm_name = vm_name
         self.parent = self.appliance.context
 
+    @property
+    def rest_obj(self):
+        return self.appliance.rest_api.collections.services.find_by(name=self.name)[0]
+
+    def add_generic_object(self, gen_obj):
```

Comment:
optional: I would name this like `add_resource_generic_obj`. As we are linking resource to `service`.